lib/modules.js

```diff
@@ -36,21 +36,20 @@ export const installBinaryModule = async ({
   repo,
   distTag,
   executable,
-  arch = os.arch(),
```

Comment:
this argument wasn't being used any more

Reply:
I am concerned about setting the arch deep inside this file. When reading the top-level post-install script, it's not clear that we may be downloading different files based on some environment variables.

Would you mind keeping `arch` here, maybe as a required argument, and setting it from env vars in the file where we are calling `getBinaryModuleExecutable`?

Reply:
good idea, like this?